index.html

```diff
@@ -219,7 +229,6 @@ <h2 class="attract">
                       Get in touch
                 </button>
         </form>
-        <hr class="line">
            </div>
 
            <div class="icons">
```

Comment:
Again, your app design is not matching the Figma template. Kindly update this section as per the given design. Also, include the horizontal line as in the original design.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/186839314-acb05ead-6544-4f58-837a-97221563cec7.png) | ![image](https://user-images.githubusercontent.com/33268927/186839405-88fe82b5-bf5e-4df2-b3de-2ef6d55f0439.png) |